### PR TITLE
Rename loan-transactions to loans-overview and move to correct section

### PR DESCRIPTION
**Pull request by igurney-codat**
# Description

Rename loan-transactions to loans-overview and move to correct section

## Type of change

Please delete options that are not relevant.

- [X] New document(s)/updating existing
- [X] Fixes
- [ ] Styling
- [ ] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: docs/assess/enhanced-liabilities/loans-overview.md [hunk 1]
```diff
@@ -1,23 +1,23 @@
 ---
-title: Loan transactions
+title: "Enhanced Liabilities - Loans"
 description: "Learn how to identify a company's loans to better understand its financial health"
-sidebar_label: "Loan transactions endpoint"
+sidebar_label: "Enhanced liabilities loans overview"
```
Comment (pzaichkina) [inline docs/assess/enhanced-liabilities/loans-overview.md:4]:
```suggestion
sidebar_label: "Overview"
```

File: docs/assess/enhanced-liabilities/loans-overview.md [hunk 2]
```diff
@@ -1,23 +1,23 @@
 ---
-title: Loan transactions
+title: "Enhanced Liabilities - Loans"
```
Comment (pzaichkina) [inline docs/assess/enhanced-liabilities/loans-overview.md:2]:
```suggestion
title: "Enhanced liabilities - loans"
```

File: sidebars/assess.js [hunk 3]
```diff
@@ -70,6 +70,7 @@ module.exports = [
     label: "Enhanced liabilties",
     collapsed: true,
     items: [
+      "assess/enhanced-liabilities/loans-overview",
       { type: "link", label: "Enhanced liabilties loan summary", href: "/assess-api#/operations/get-loan-summary" },
```
Comment (pzaichkina) [inline sidebars/assess.js:74]:
```suggestion
      { type: "link", label: "Loan summary", href: "/assess-api#/operations/get-loan-summary" },
```

File: sidebars/assess.js [hunk 4]
```diff
@@ -70,6 +70,7 @@ module.exports = [
...
       { type: "link", label: "Enhanced liabilties loan transactions", href: "/assess-api#/operations/list-loan-transactions" },
```
Comment (pzaichkina) [inline sidebars/assess.js:75]:
```suggestion
      { type: "link", label: "Loan transactions", href: "/assess-api#/operations/list-loan-transactions" },
```

File: docs/assess/enhanced-liabilities/loans-overview.md [hunk 5]
```diff
@@ -1,23 +1,23 @@
...
 Loan data is often not uniform, spread across a variety of software applications and the data can be difficult to interpret. 
 These loans are often hard to identify, analyze, and group.
-Manually going through data and identifying loans is time-consuming and repetitive, and introduces room for human error.
+Manually going through data and identifying loans is time-consuming and repetitive, and introduces the chance for human error.
 
 Our Loan Transactions endpoint can identify loans from a company's accounting, banking, and commerce integrations. 
 It can work across a variety of integrations for all three integration types.
 
-The endpoint will give analysts a better idea of the loan obligations a company may have by identifying their loan transactions and loan repayments.
+These endpoints will give analysts a better idea of the loan obligations a company may have by identifying their loan transactions and loan repayments.
```
Comment (pzaichkina) [inline docs/assess/enhanced-liabilities/loans-overview.md:14]: so this paragraph has been changed to say "these endpoints", but the rest of the page still talks about a single Loan transactions endpoint. does the rest of it need to be changed also?

**Review by pzaichkina — CHANGES_REQUESTED** [review]

File: docs/assess/enhanced-liabilities/loans-overview.md (hunk 5 above)
Comment (igurney-codat) [reply docs/assess/enhanced-liabilities/loans-overview.md:14]: Yes it should be plural as there are two endpoints - loan transactions and loan summary

File: sidebars/assess.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (igurney-codat) [reply sidebars/assess.js]: I agree and I have accepted the changes

**Comment by pzaichkina** [comment]
Hi @iangurney,

Thank you for addressing the comments. As you are aware, we are currently in the docs freeze, and unfortunately, the changes you are planning to make will result in a conflict for us.

You have two options:
* Closing this PR and opening another one after the freeze is done to make these changes. This means you will contribute to our `main` branch after it has been overhauled.
* Closing this PR and opening another one to contribute to the **`fix-typos` branch instead of** the `main` branch. This means you will contribute to the overhaul branch, and your changes will go live together with the overhaul itself.

If you need any guidance on either action, give me a shout.

Thanks,
Polina

**Pull request closed without merging**